python/freetensor/core/remote_task_scheduler.py

```diff
@@ -0,0 +1,735 @@
+from __future__ import annotations
+import queue
+import threading
+import time
+import copy
+from typing import Any, List
+from typing import Dict
+'''
+task_type is specified
+1.search_task
+2.measure_task
+(minus means result)
+special param:
+0.other(transfer control)
+then check param
+    trans_c:(those are conbined with additional param: time_stamp)
+    0. logger
+    1. broadcast availability
+    2. return inavailability
+    3. new_task_request    
+minus means receive
+'''
+
+
+class Task(object):
+    src_server_uid: str
+    #
+    target_server_uid: str
+    #
+    task_uid: int
+    submit_uid: int
+    #
+    params: Any
+    #
+    task_type: int
+    #
+    block_start: int = 0
+    block_end: int = 6400
+    #
+    normal_bundle_size: int = 4  # the number of tasks in a bundle
+    min_bundle_size: int = 26  #the minimum size of a bundle(in blocks)
+    single_task_block_size: int = 100  # the size of single task(in blocks)
+    max_block_to_split: int = 1600  #no larger than the number of blocks will be further splitted
+
+    #
+
+    def split(self, _block_start: int, _block_end: int) -> Task:
+        r = copy.copy(self)
+        r.block_start = _block_start
+        r.block_end = _block_end
+        return r
+
+    def run(self) -> None:
+        print(self.block_start)
+        time.sleep(1)
+        print(self.block_end)
```

Comment:
Please merge with other branches first if the functions are incomplete.

Reply:
Ok